### PR TITLE
feat(Harvest): Add website link in connector data

### PR DESCRIPTION
**Pull request by doubleface**
This will make it easier to navigate from cozy-home

Screenshot_2020-08-27 Cozy Home

### CODE REVIEW

File: packages/cozy-harvest-lib/src/components/cards/WebsiteLinkCard.jsx [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+  const linkStyle = { textTransform: 'lowercase', margin: 0 }
+  const hrStyle = { color: palette['silver'] }
+  const label = new URL(link).host
+
+  return (
+    <Card>
+      <SubTitle className="u-mb-1">{t('card.websiteLink.title')}</SubTitle>
+      <hr style={hrStyle} />
+      <Media align="top">
+        <Img style={imgStyle}>
```
Comment (Crash--) [inline packages/cozy-harvest-lib/src/components/cards/WebsiteLinkCard.jsx:24]:
```suggestion
        <Img className={'u-pr-1"}>
```

File: packages/cozy-harvest-lib/src/components/cards/WebsiteLinkCard.jsx [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+const WebsiteLinkCard = ({ link, t }) => {
+  const imgStyle = { paddingRight: '1rem' }
+  const linkStyle = { textTransform: 'lowercase', margin: 0 }
+  const hrStyle = { color: palette['silver'] }
+  const label = new URL(link).host
+
+  return (
+    <Card>
+      <SubTitle className="u-mb-1">{t('card.websiteLink.title')}</SubTitle>
+      <hr style={hrStyle} />
```
Comment (Crash--) [inline packages/cozy-harvest-lib/src/components/cards/WebsiteLinkCard.jsx:22]:
```suggestion
      <hr className={"u-silver"} />
```

File: packages/cozy-harvest-lib/src/components/cards/WebsiteLinkCard.jsx [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+          <Icon icon="globe" color={palette['coolGrey']} />
+        </Img>
+        <Bd>
+          <ButtonLink
+            subtle
+            target="_blank"
+            href={link}
+            label={label}
+            theme="text"
+            className="u-dodgerBlue"
```
Comment (Crash--) [inline packages/cozy-harvest-lib/src/components/cards/WebsiteLinkCard.jsx:34]:
```suggestion
            className="u-dodgerBlue u-m-0"
```

File: packages/cozy-harvest-lib/src/components/cards/__snapshots__/WebsiteLinkCard.spec.jsx.snap [hunk 4]
```diff
@@ -0,0 +1,7 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`WebsiteLinkCard should render 1`] = `
+<withI18n(WebsiteLinkCard)
+  link="https://fc.assure.ameli.fr/FRCO-app/login"
+/>
+`;
```
Comment (Crash--) [inline packages/cozy-harvest-lib/src/components/cards/__snapshots__/WebsiteLinkCard.spec.jsx.snap:7]: Not sure this is the withI18n we want to snapshot. Not very useful I think. Snapshoting WebsiteLinkCard will be better

**Review by Crash-- — COMMENTED** [review]
Cool to see you working on UI :D. 

You should avoid using inline style. UI comes with a lot of utility classes (https://docs.cozy.io/cozy-ui/styleguide/section-utilities.html) and we should use it. 

If there is no utility class for want you want (for instance textTransform lowercase...) you should create a new class

File: packages/cozy-harvest-lib/src/components/cards/__snapshots__/WebsiteLinkCard.spec.jsx.snap (hunk 4 above)
Comment (doubleface) [reply packages/cozy-harvest-lib/src/components/cards/__snapshots__/WebsiteLinkCard.spec.jsx.snap:7]: This is a copy from some other harvest unit test. It allow me to force a locale. It seems that passing t as a property in the unit test is not enough.

Comment (Crash--) [reply packages/cozy-harvest-lib/src/components/cards/__snapshots__/WebsiteLinkCard.spec.jsx.snap:7]: Let's do that in an other way. 

Remove the translate() HOC and use our new useI18n() hook. (If we can use our new method in our new component, it's better ;)) 

Your component will be: 
```
const WebsiteLinkCard = ({ link}) => {
  const linkStyle = { textTransform: 'lowercase' }
  const label = new URL(link).host
  const { t } = useI18n()
  return (
    <Card>
      <SubTitle className="u-mb-1">{t('card.websiteLink.title')}</SubTitle>
      <hr className="u-silver" />
      <Media align="top">
        <Img className="u-pr-1">
          <Icon icon="globe" color={palette['coolGrey']} />
        </Img>
        <Bd>
          <ButtonLink
            subtle
            target="_blank"
            href={link}
            label={label}
            theme="text"
            className="u-dodgerBlue u-m-0"
            style={linkStyle}
          />
          <Caption>{t('card.websiteLink.description')}</Caption>
        </Bd>
      </Media>
    </Card>
  )
}
```

Then I will only test a few things like : 
```
const wrapper = mount(
      <I18n dictRequire={() => enLocale} lang="en">
        <WebsiteLinkCard link="https://fc.assure.ameli.fr/FRCO-app/login" />
      </I18n>
    )
    const component = wrapper.find(WebsiteLinkCard)
    expect(component.find('ButtonLink').props().href).toEqual(
      'https://fc.assure.ameli.fr/FRCO-app/login'
    )
    expect(component.find('ButtonLink').props().label).toEqual(
      'fc.assure.ameli.fr'
    )
```

to be sure that we have the right href & label.


Having a snapshot with `<withI18n(WebsiteLinkCard)
  link="https://fc.assure.ameli.fr/FRCO-app/login"
/>` is not wanted.

If you really want to have the full snapshoted component you can either mount() and then component.debug().match... or use a serializer like https://github.com/adriantoine/enzyme-to-json or something similar

**Pull request merged**